ui/src/overridden/frontsite/Document/DocumentDetails/DocumentMetadataTabs/Identifiers.js

```diff
@@ -0,0 +1,211 @@
+import {
+  InfoPopup,
+  SeparatedList,
+  vocabularyApi,
+  withCancel,
+} from '@inveniosoftware/react-invenio-app-ils';
+import capitalize from 'lodash/capitalize';
+import _isEmpty from 'lodash/isEmpty';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { Divider, Table, Loader } from 'semantic-ui-react';
+
+export class Identifiers extends Component {
```

Comment:
This is not idea because if we want to display this link somewhere else, e.g. backoffice, we need to do the same thing, fetch from the backend for each vocab.
I am also not convinced by the fact that the UI should know that for this specific vocab, the URL exists and we need to fetch the vocab definition.
At the same time I don't have better ideas. 
ping @kprzerwa @zzacharo

Reply:
the alternative is to hardcode the urls in the config like initially proposed, however it will require placing some conditional statements, also depending on the vocabularies, so it also creates similar dependency

Reply:
It does not: the dependency is a config, that can be changed.
The UI will simply use a generic config which, if set, will render the identifier as a link. There is no hardcoding in the code, it is config.